DEVDOCS-5045: [update] Remove non-token oauth scopes from token creation docs (#95)
bigcommerce/docs: docs/api-docs/storefront/graphql/graphql-api-overview.mdx

### Creating a token

> Use the [Create a storefront token](/docs/rest-authentication/tokens#create-a-token) REST endpoint to request JWT bearer tokens that authenticate requests to the GraphQL Storefront API. Use a [token creation scope](/docs/start/authentication/api-accounts#token-creation-scopes) for the [store-level or app-level API account](/docs/start/authentication/api-accounts) in the request.

Reviewer: ```suggestion
Use the [Create a storefront token](/docs/rest-authentication/tokens#create-a-token) REST endpoint to request JWT bearer tokens that authenticate requests to the GraphQL Storefront API. Add a [token creation scope](/docs/start/authentication/api-accounts#token-creation-scopes) to the [store-level or app-level API account](/docs/start/authentication/api-accounts) you use to generate tokens.
```

Author: I had it like this before and changed it.

### Customer impersonation tokens

> It's also possible to generate tokens for use in server-to-server or headless interactions with a trusted consumer using the [Create a customer impersonation token](/docs/rest-authentication/tokens/customer-impersonation-token) endpoint. Use the Storefront API Customer Impersonation Tokens [OAuth scope](/docs/rest-authentication/tokens/customer-impersonation-token#create-a-token) in the request.

Reviewer: ```suggestion
You can use Customer Impersonation Tokens to authenticate requests to the GraphQL Storefront API in server-to-server and headless interactions. Add the [Storefront API Customer Impersonation Tokens scope](/docs/start/authentication/api-accounts#token-creation-scopes) to the [store-level or app-level API account](/docs/start/authentication/api-accounts) you use when you [Create a customer impersonation token](/docs/rest-authentication/tokens/customer-impersonation-token). 
```
